Lesson_7/ICoder.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Lesson_7;
+
+public interface ICoder
+{
+    public string Encode(string s);
+    public string Decode(string s);
```

Comment:
Внутри определения интерфейса всё является публичным. Можно модификаторы не писать.
```suggestion
    string Encode(string s);
    string Decode(string s);
```